assignments/callbacks.js

```diff
@@ -27,20 +27,46 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  cb(arr.length);
 }
+function printLength(lengthOfArr) {
+  console.log(lengthOfArr);
+}
+
+getLength(items, printLength);
```

Comment:
I like seeing the separation of the `cb` here into its own function rather than just write it #36 as an argument.